src/lib/sitemap/docs-content-fields.ts

```diff
@@ -6,14 +6,45 @@
 import { makeSitemapField } from './helpers'
 
 export async function allDocsFields() {
-	const getDocsPaths = await fetch(
+	const getContentAPIDocsPaths = await fetch(
 		`${process.env.MKTG_CONTENT_DOCS_API}/api/all-docs-paths`
 	)
-	const { result: docsResult } = await getDocsPaths.json()
-	return docsResult.map((page: { path: string; created_at: string }) =>
-		makeSitemapField({
-			slug: `${page.path}`,
-			lastmodDate: page.created_at,
-		})
+	const { result: contentAPIDocsResult } = await getContentAPIDocsPaths.json()
+
+	// If there are docs that have been migrated to the unified docs repo, get the paths for those docs
+	// and merge them with the paths for the docs that haven't been migrated from the content API
+	if (__config.flags?.unified_docs_migrated_repos.length > 0) {
+		const getUDRDocsPaths = await fetch(
+			`${process.env.UNIFIED_DOCS_API}/api/all-docs-paths`
+		)
+		const { result: udrDocsResult } = await getUDRDocsPaths.json()
+
+		// Filter the result from the content API to take out the duplicate paths between
+		// the content API and UDR
+		const filteredContentAPIDocsResult = contentAPIDocsResult.filter(
```

Comment:
@LeahMarieBush We should only merge in the paths from the products that are in `unified_docs_migrated_repos`. Cause `/api/all-docs-paths` might return products/paths that are not in `unified_docs_migrated_repos`. (Like when we are testing a new product repo in UDR.)

Reply:
Also what do you think should we add `productSlugs` as am optional param to /api/all-docs-paths? That way we can do the filtering on the API server and then do a simple merge here.

Reply:
Yeah I think that adding that optional param to the endpoint would make this a lot easier, then we can just fetch the paths needed. I'll get a story written for that and put this on hold until that ticket is complete